### PR TITLE
[storage] Add parallel uploads to the limit usage

### PR DESCRIPTION
**Pull request by shomix**
This PR will:
- remove the `usedStorageFileCoin` from the limit interface ( this doesn't exists )
- add `usedParallelUploads` and `parallelUploadsLimit` to the limit.
- remove the compute limits from the `storage` package because they are not applicable  for the web-app organizaiton.

### CODE REVIEW

**Review by dusanstanis — DISMISSED** [review]

File: packages/storage/src/bucket-manager/interfaces.ts [hunk 1]
```diff
@@ -37,4 +37,29 @@ interface Upload {
   protocol: string;
 }
 
-export { Bucket, Domain, Upload, BucketStateEnum, UploadStatusEnum };
+interface UsageWithLimits {
```
Comment (aayushmahapatra) [inline packages/storage/src/bucket-manager/interfaces.ts:40]: Is everything optional in this interface? 🤔

**Review by aayushmahapatra — APPROVED** [review]
Just one question.

File: packages/storage/src/bucket-manager/interfaces.ts (hunk 1 above)
Comment (shomix) [reply packages/storage/src/bucket-manager/interfaces.ts:40]: Removed the ? and added default values when mapping

**Review by aayushmahapatra — APPROVED** [review]
Lgtm

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**